e2e/schematics/ngrx.test.ts

```diff
@@ -11,6 +11,7 @@ describe('ngrx', () => {
       runCLI(
         'generate ngrx users --module=apps/myapp/src/app/app.module.ts --root --collection=@nrwl/schematics'
       );
+      copyMissingPackages();
```

Comment:
why is this needed?

Reply:
This is needed because generating ngrx installs packages and therefore wipes the @nrwl dependencies. This copies it back